ghost/core/core/server/data/migrations/versions/5.31/2023-01-19-07-46-add-mentions-table.js

```diff
@@ -0,0 +1,23 @@
+const {addTable} = require('../../utils');
+
+// stay below 
+const mentionsTable = addTable('mentions', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    source: {type: 'string', maxlength: 2000, nullable: false},
+    source_title: {type: 'string', maxlength: 2000, nullable: true},
+    source_site_title: {type: 'string', maxlength: 2000, nullable: true},
+    source_excerpt: {type: 'string', maxlength: 2000, nullable: true},
+    source_author: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Maybe `2000` is a bit over the top for author? Our internal limit for author names is `191` for our own posts

Reply:
But not so important I think 🤷‍♂️

Reply:
+ you followed the spec above